concepts/05 UI Components/Splitter/20 Implement Custom Scrolling/00 Implement Custom Scrolling.md

```diff
@@ -0,0 +1,222 @@
+Splitter displays native scroll bars when a pane content is smaller than the pane itself (for instance, as a result of resize).
+
+You can replace native scroll bars with DevExtreme ScrollView component, which allows you to additionally configure scrolling. For example, you can specify [when to show a scroll bar](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/#showScrollbar). For more information about the available options, refer to the [ScrollView API section](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/).
```

Comment:
I'm not sure we should talk about "native scroll bars" as equivalent to "ScrollView component" (as in "replace native scroll bars with ScrollView"). The component is a container that hosts some content and also displays scrollbars. So I wouldn't say that you can just replace one with the other. 

```suggestion
You can hide native scroll bars and use a DevExtreme ScrollView component. This component includes extended scroll bar settings. For example, you can use different [scroll bar visibility modes](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/#showScrollbar). For more information about the available options, refer to the [ScrollView API section](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/).
```